metadata-analysis/Metadata-Analysis.Rmd

```diff
@@ -217,13 +389,160 @@ Eva did not begin tracking downloads until December 29, 2022.
 
 counts_by_week %>%
```

Comment:
this doesn't show correctly for me, the only values in the WeekEnding column in the filtered dataframe are both 7/22/2024 so I think the plot just doesn't know what to do with it

Reply:
fixed